src/instrumentations/redis/RedisInstrumentation.ts

```diff
@@ -0,0 +1,23 @@
+import { CommonUtils, ScrubContext } from '@lumigo/node-core';
+import { RedisInstrumentation } from '@opentelemetry/instrumentation-redis-4';
+import { getSpanAttributeMaxLength } from '../../utils';
+import { Instrumentor } from '../instrumentor';
+
+export default class LumigoRedisInstrumentation extends Instrumentor<RedisInstrumentation> {
+  getInstrumentedModule(): string {
+    return 'redis';
+  }
+
+  getInstrumentation(): RedisInstrumentation {
+    return new RedisInstrumentation({
+      dbStatementSerializer: function (cmdName, cmdArgs) {
```

Comment:
this will include a `db.statement` attribute. we can also use the responseHook, but the results are identical (the only difference being that we could rename the attribute, but i'm assuming that `db.statement` is the standard approach.